### PR TITLE
DXCDT-339: Fix issue with properties not being computed on oidc connections

### PR DESCRIPTION
**Pull request by sergiught**
### 🔧 Changes

The following option fields on the connection `authorization_endpoint`,  `issuer`,  `jwks_uri`,  `token_endpoint`, or `user_info_endpoint` can be set to default values by the API, thus we need to make them Computed in order to prevent erroneous diffs when planning or applying. 

### 📝 Checklist

- [x] All new/changed/fixed functionality is covered by tests (or N/A)
- [x] I have added documentation for all new/changed functionality (or N/A)

### CODE REVIEW

File: internal/provider/resource_auth0_connection.go [hunk 1]
```diff
@@ -23,6 +23,7 @@ var connectionSchema = map[string]*schema.Schema{
 	"display_name": {
 		Type:        schema.TypeString,
 		Optional:    true,
+		Computed:    true,
```
Comment (Widcket) [inline internal/provider/resource_auth0_connection.go:26]: > The following option fields on the connection authorization_endpoint, issuer, jwks_uri, token_endpoint, or user_info_endpoint can be set to default values by the API

`display_name` as well?

Comment (willvedd) [reply internal/provider/resource_auth0_connection.go:26]: Apparently `display_name` is a computed value for OIDC connection types, defaulting to the `name` field if not defined. It's a bit of a toss-up of wether its' worth it to mark as computed for all connections to accommodate this single connection type, but the consequences are minimal either way. @sergiught and I just changed our minds on this and decided to remove to minimize the effects for the majority of users.

Comment (willvedd) [inline internal/provider/resource_auth0_connection.go:26]:
```suggestion
```

**Review by willvedd — APPROVED** [review]
Makes complete sense now that OIDC discovery has been implemented on the server-side.

**Review by Widcket — APPROVED** [review]

**Pull request merged**